README.md

```diff
@@ -1,15 +1,12 @@
 [![Community Extension](https://img.shields.io/badge/Community%20Extension-An%20open%20source%20community%20maintained%20project-FF4700)](https://github.com/camunda-community-hub/community) ![Compatible with: Camunda Platform 8](https://img.shields.io/badge/Compatible%20with-Camunda%20Platform%208-0072Ce) [![](https://img.shields.io/badge/Lifecycle-Stable-brightgreen)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#stable-)
 
-# Spring-Zeebe -> Camunda Spring Zeebe SDK
+# Spring-Zeebe
 
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.camunda/spring-zeebe/badge.svg)](https://maven-badges.herokuapp.com/maven-central/io.camunda/spring-zeebe)
 [![Project Stats](https://www.openhub.net/p/spring-zeebe/widgets/project_thin_badge.gif)](https://www.openhub.net/p/spring-zeebe)
 
 > [!IMPORTANT]
-> This is the community maintained version of the Spring Zeebe SDK. For the officially Camunda supported version of this SDK, please refer to the [Zeebe repository](https://github.com/camunda/zeebe).
-
-
-
+> This is a community maintained project. For the fully supported Camunda Spring SDK, refer to the [Zeebe repository](https://github.com/camunda/zeebe). For public information about the SDK, see [this issue](https://github.com/camunda/issues/issues/711).
```

Comment:
```suggestion
> This is a community maintained project. For the fully supported Camunda Spring Zeebe SDK, refer to the [official documentation](https://docs.camunda.io/docs/next/apis-tools/spring-zeebe-sdk/getting-started/).
```

Reply:
Users are asking for more context because these experiences aren't at parity yet. 

Instead of linking to the Zeebe repo, we can link to the docs, but I think we should keep the link to the public issue too.

Reply:
Got it. I didn't see that context. Can you please propose then how to frame it with the link to the docs? Thanks

Reply:
Will incorporate @aleksander-dytko's suggestion via commit. Resolving this conversation and will add a proposal on the new commit content.